src/WalletInit/SaveNanoX/SaveNanoXScreen.tsx

```diff
@@ -13,10 +17,19 @@ export const SaveNanoXScreen = () => {
   const route = useRoute<RouteProp<WalletInitRoutes, 'save-nano-x'>>()
   const {networkId, walletImplementationId, hwDeviceInfo} = route.params
 
+  const intl = useIntl()
   const {createWallet, isLoading} = useCreateBip44Wallet({
     onSuccess: () => {
       resetToWalletSelection()
     },
+    onError: (error) => {
+      InteractionManager.runAfterInteractions(() => {
+        const newLocal = error instanceof NetworkError
+        return newLocal
+          ? showErrorDialog(errorMessages.networkError, intl)
+          : showErrorDialog(errorMessages.generalError, intl, {message: error.message})
```

Comment:
Do we need `newLocal`?
```suggestion
        return error instanceof NetworkError
          ? showErrorDialog(errorMessages.networkError, intl)
          : showErrorDialog(errorMessages.generalError, intl, {message: error.message})
```

Reply:
vscode autorefactor...

Reply:
so...no

Reply:
it doesn't make sense to me, running on my VSCode the result is even different: `
      const newLocal = error instanceof NetworkError
        ? showErrorDialog(errorMessages.networkError, intl)
        : showErrorDialog(errorMessages.generalError, intl, { message: error.message })
      InteractionManager.runAfterInteractions(() =>
        newLocal
      )
`